python/ray/dataframe/utils.py

```diff
@@ -465,4 +465,11 @@ def _correct_column_dtypes(*column):
     Args:
     """
     concat_column = pd.concat(column, copy=False)
-    return create_blocks_helper(concat_column, len(column), 1)
+    partitions = []
```

Comment:
Can this not be fixed in `create_blocks_helper`? It looks like that would be the source of the problem.